pro/src/ui-kit/form/DurationInput/DurationInput.tsx

```diff
@@ -0,0 +1,55 @@
+import { useField } from 'formik'
+import React from 'react'
+
+import TextInput from '../TextInput'
+
+import { parseMinutesToHours } from './utils/parseMinutesToHours'
+
+export interface IDurationInputProps {
+  label: string
+  name: string
+  isOptional?: boolean
+  className?: string
+}
+
+const DurationInput = ({
+  label,
+  name,
+  isOptional = false,
+  className,
+  ...props
+}: IDurationInputProps): JSX.Element => {
+  const [field, meta, helpers] = useField({ name })
+
+  const onDurationBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    const [updatedHours, updatedMinutes] = event.target.value.split(':')
+    const updatedDurationInMinutes =
+      parseInt(updatedHours || '0') * 60 + parseInt(updatedMinutes || '0')
+    const durationMinutes = parseMinutesToHours(updatedDurationInMinutes)
+
+    helpers.setValue(durationMinutes)
+  }
+
+  const onDurationChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const durationInputValue = event.target.value.match(/\d*:?[0-5]?\d?/)
+    !meta.touched &&
+      durationInputValue &&
+      helpers.setValue(durationInputValue[0])
+  }
```

Comment:
verifier que le tableau `durationInputValue` contient au moins une valeurs devrait satisfaire TypeScript :)
```suggestion
    const durationInputValue = event.target.value.match(/\d*:?[0-5]?\d?/)
    durationInputValue && 
      durationInputValue.length > 0 &&
      helpers.setValue(durationInputValue[0])
  }
  }
```